inbox/mailsync/backends/gmail.py

```diff
@@ -598,3 +430,172 @@ def g_msgids(namespace_id, session, in_):
         .all()
     )
     return {g_msgid for g_msgid, in query}
+
+
+class GmailSyncMonitor(ImapSyncMonitor):
+    sync_engine_class: ClassVar[type[FolderSyncEngine]] = GmailFolderSyncEngine
```

Comment:
And this code was moved below with

`sync_engine_class: ClassVar[type[FolderSyncEngine]] = GmailFolderSyncEngine`

moved from the initializer to class body.